.github/workflows/pages.yml

```diff
@@ -14,6 +14,8 @@ jobs:
           static_site_generator: next
       - uses: ./.github/actions/prepare
       - run: pnpm build
+        env:
+          CUSTOM_BASE_PATH: "/opencontributionssite"
```

Comment:
Heh, sorry, I rewrote the Git history on the `main` branch to remove these lines. Your branch must be based on the old version.

```suggestion
```